azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -80,14 +81,48 @@ async def ensure_deploy_key_secret(client, cluster: cluster_crd.Cluster):
     return base64.b64decode(secret.data["id_ed25519.pub"]).decode()
 
 
+async def get_global_extravars(client):
```

Comment:
Can we please get some tests around this bit? It looks pretty magic, and worth some tests.